libs/community/langchain_community/chat_models/reka.py

```diff
@@ -329,11 +330,29 @@ async def _agenerate(
 
         return ChatResult(generations=[ChatGeneration(message=message)])
 
-    def get_num_tokens(self, text: str) -> int:
+    def get_num_tokens(self, input: Union[str, BaseMessage, List[BaseMessage]]) -> int:
```

Comment:
looks like the input type here is not matching the one in the `token_counter` callable (failing linter)